### PR TITLE
BUG: Remove email directory from workflows for unit testing

### PR DESCRIPTION
**Pull request by meoflynn**
### Description: 
Having a directory named email breaks the debug mode in PyCharm as it has the same name as the email python package. Removed the email directory and moved files to be under workflows only.

### Special Notes:

---

### Submitter:

Have you (where applicable):

* [ ] Added unit tests?
* [ ] Checked the latest commit runs on Dev?
* [ ] Updated the example config file(s) and README?

---

### Reviewer

Does this PR:

* [ ] Place non-StackStorm code into the `lib` directory?
* [ ] Have unit tests for the action/sensor and `lib` layers?
* [ ] Have clear and obvious action parameter names and descriptions?

### CODE REVIEW

**Review by anish-mudaraddi — CHANGES_REQUESTED** [review]
Why remove submodule entirely instead of just renaming it to `email_workflows` or something? 

I used submodules so we can better4 organise workflows

File: tests/lib/workflows/test_send_decom_flavor_email.py
[the remark this answers is not in the document]
Comment (anish-mudaraddi) [reply tests/lib/workflows/test_send_decom_flavor_email.py:1]: sorry - this is for the file `/test/actions/test_workflow_actions`

**Comment by anish-mudaraddi** [comment]
It's interesting your Pycharm breaks due to the directory being named `email` - there's another directory named email - `lib/structs/email` - maybe you should rename that one in the PR too?

File: tests/actions/test_workflow_actions.py [hunk 1]
```diff
@@ -1,27 +1,9 @@
...
-    workflow_module = __import__(f"workflows.{submodule_name}.{action_name}")
-    assert hasattr(workflow_module, f"{submodule_name}")
-    submodule = getattr(workflow_module, f"{submodule_name}")
-
-    assert hasattr(submodule, action_name)
-    action_module = getattr(submodule, action_name)
-
-    assert hasattr(action_module, action_name)
-
-
```
Comment (anish-mudaraddi) [inline tests/actions/test_workflow_actions.py:24]: you should still keep this test - just remove the submodule part

```python
@pytest.mark.paramterize("action_name", ["send_decom_flavor_email"])
def test_module_exists(action_name):
    """
    Test that each action's entry-point module exists 
    """
    workflow_module = import_module(f"workflows.{action_name}")
    assert hasattr(workflow_module, action_name)
    action_module = getattr(workflow_module, action_name)
    
    # this assumes that for each workflow file, the function the action runs has the same name as the file - you could hardcode this to `main` if you want
    assert hasattr(action_module, action_name)
    # assert hasattr(action_module, "main") # if you choose to hard-code it to main

```

**Review by anish-mudaraddi — CHANGES_REQUESTED** [review]
I believe @DavidFair mentioned maybe enforcing that each workflow file has a `main()` function that is called by the action first. 

Right now I have it so that each workflow file contains a function with the same name as the file which is run - I agree with @DavidFair  that we should use `main` instead. Worth changing that in this PR too?

**Comment by meoflynn** [comment]
I think maybe we create a PR for updating workflows to have a `main()` method

**Review by anish-mudaraddi — APPROVED** [review]

**Review by DavidFair — APPROVED** [review]

**Pull request merged**